index.html

```diff
@@ -4,464 +4,524 @@
                 <img
-                  class="google-img"
-                  src="./src/icons/flat-color-icons_google.svg"
-                  alt="google icon"
+                  class="search-container__search-icon"
+                  src="./src/icons/search-icon.svg"
+                  alt="search"
                 />
-              </div>
 
-              <div class="right-wrapper__flying-icon paypal-icon">
-                <img
-                  class="paypal-img"
-                  src="./src/icons/logos_paypal.svg"
-                  alt="paypal icon"
+                <input
+                  class="search-input"
+                  type="text"
+                  placeholder="Job title or Keyword"
+                  name="text"
+                  value=""
                 />
               </div>
 
-              <div class="right-wrapper__flying-icon indeed-icon">
+              <span class="line"></span>
+
+              <div class="search-area">
                 <img
-                  class="indeed-img"
-                  src="./src/icons/simple-icons_indeed.svg"
-                  alt="indeed icon"
+                  class="search-container__location-icon"
+                  src="./src/icons/location-icon.svg"
+                  alt="location"
                 />
-              </div>
 
-              <div class="right-wrapper__flying-icon slack-icon">
-                <img
-                  class="slack-img"
-                  src="./src/icons/grommet-icons_slack.svg"
-                  alt="slack icon"
+                <input
+                  class="location-input"
+                  type="text"
+                  placeholder="Location"
+                  name="text"
+                  value=""
                 />
               </div>
 
-              <!-- First flying card -->
-              <div class="right-wrapper__flying-card first-flying-card">
-                <div class="flying-card__img-block first-img">
-                  <div class="flying-card__circle">
-                    <img src="./src/icons/star.svg" alt="star" />
-                    <p class="flying-card__rating">4.8</p>
-                  </div>
-                </div>
-                <h2 class="flying-card__title">Maliha</h2>
-                <p class="flying-card__subtitle">UX Designer</p>
+              <button
+                type="submit"
+                class="buttons buttons__search-button"
+                href="#"
+              >
+                Search
+              </button>
+            </form>
+
+            <!-- Search form -->
+
+            <!-- <form class="search-container__location-form" method="get">
+
+            </form> -->
+          </div>
+        </div>
+
+        <!-- Graphics -->
+        <div class="graphics">
+          <img
+            class="graphics__img"
+            src="./src/img/satisfied-lovely-woman-holds-modern-cell-phone-1.png"
+            alt="satisfied-lovely-woman-holds-modern-cell-phone"
```

Comment:
```suggestion
            alt="A satisfied lovely woman holds a modern cell phone"
```
The "alt" string should be written grammatically correctly and with whitespaces, not dashes.

Reply:
renamed alt text